pkg/alerts/example_policy_test.go

```diff
@@ -10,10 +10,9 @@ import (
 func Example_policy() {
 	// Initialize the client configuration.  A Personal API key or Admin API key
 	// is required to communicate with the backend API.
-	cfg := config.Config{
-		AdminAPIKey:    os.Getenv("NEW_RELIC_ADMIN_API_KEY"),
-		PersonalAPIKey: os.Getenv("NEW_RELIC_API_KEY"),
-	}
+	cfg := config.New()
+	cfg.AdminAPIKey = os.Getenv("NEW_RELIC_ADMIN_API_KEY")
+	cfg.PersonalAPIKey = os.Getenv("NEW_RELIC_API_KEY")
```

Comment:
Is there a reason here not to have the `New()` call populate the API key fields?  If not, I might suggest this as a change, so that a a new user only need pay attention to `client := New(config.New())` to have a fully working config.

Reply:
`New()` should not read from the Env, as what if you have a config you want and happen to have env vars set?  (Common use case when I'm doing dev work and switch to a demo real quick).

Having a `NewFromEnv` or something along those lines was a thought, though did not implement at this time.